034-instanced-pipelines/proposal.md

```diff
@@ -0,0 +1,75 @@
+# Summary
+
+Instanced pipelines group together pipelines which share a common template configured with different ((vars)). They provide a simple two-level hierarchy and automatic archiving of instances which are no longer needed.
+
+# Proposal
+
+Pipelines can be configured with 'instance vars' like so:
+
+```yaml
+fly set-pipeline -p branch --instance-var branch=feature/foo
+```
+
+This will configure a `branch` pipeline, with the `((branch))` var set to `"feature/foo"`.
+
+Instance vars are used as part of the pipeline identifier in the UI and API. There can be multiple instances of a pipeline with the same name:
+
+```yaml
+fly set-pipeline -p branch --instance-var branch=feature/foo
+fly set-pipeline -p branch --instance-var branch=feature/bar
+```
+
+Instanced pipelines sharing the same name will be grouped together in the web UI.
```

Comment:
This is great! Some sort of hierarchical display would be awesome. https://buildpacks.ci.cf-app.com/ has lots of "instanced" pipelines we create through `erb.yml` craziness, and being able to visually organize them better would be super!